### DISCUSSION TITLE
Unable to start Valheim LGSM server

### DISCUSSION THREAD

**Discussion by hirschbergt** [GameServerManagers/LinuxGSM#3358] [General]
I followed all the steps to install the Valheim LGSM server and dependencies, but when I try starting the server it does not seem to be running properly. When I use ./vhserver details it says "ONLINE," but when I check the vhserver-script log file I see the following:

Mar 11 12:44:41.417 vhserver: START: PASS: Started Valhallas Armpit
Mar 11 12:44:41.433 vhserver: START: INFO: LinuxGSM version: v21.1.3
Mar 11 12:44:41.434 vhserver: START: PASS: core_exit.sh exiting with code: 0
Mar 11 12:46:25.005 vhserver: STOP: INFO: Graceful: CTRL+c
Mar 11 12:46:30.551 vhserver: STOP: PASS: Graceful: CTRL+c: OK: 4 seconds
Mar 11 12:46:30.573 vhserver: STOP: INFO: LinuxGSM version: v21.1.3
Mar 11 12:46:30.575 vhserver: STOP: PASS: core_exit.sh exiting with code: 0
Mar 11 12:46:35.260 vhserver: DETAILS: INFO: LinuxGSM version: v21.1.3
Mar 11 12:46:35.261 vhserver: DETAILS: PASS: core_exit.sh exiting with code: 0
Mar 11 12:47:25.771 vhserver: START: INFO: Using anonymous Steam login
Mar 11 12:47:26.444 vhserver: START: INFO: Rotating log files

So I really cannot tell if the server is even running. Any thoughts?

**Comment by rconjoe** [comment]
Any logs from the console, and ./vhserver details?

**Comment by HeinzM** [comment]
Did you set a passwort? If so, try to use a longer one. It needs at least five characters.
And please check portforwarding.
